selenium_tests/homepage/test_search.py

```diff
@@ -0,0 +1,27 @@
+import time
+
+from selenium_tests.base import BaseSeleniumTests
+from home.factories import (
+    SectionIndexFactory,
+    SectionFactory,
+    ArticleFactory
+)
+
+from selenium_tests.pages import SearchPage
+
+class SearchSeleniumTests(BaseSeleniumTests):
+
+    def setUp(self):
+        self.setup_blank_site()
+        self.sections_index = SectionIndexFactory(parent=self.site.root_page)
+        self.section01 = SectionFactory(parent = self.sections_index)
+        self.article01 = ArticleFactory(parent=self.section01, title = 'Test article for search - asdf;lkj')
+
+    def test_search_bar(self):
+
+        home_page = self.visit_page(self.home)
+        home_page.small_search_button_select()
+        search_page = SearchPage(self.selenium)
+        search_page.search('asdf;lkj')
```

Comment:
There should be some sort of assertion to verify whether the search was successful (or not).

Reply:
As previously discussed, currently getting an error when I try to view the search results. Not an error on the site itself but something going on with the way we are testing. @sheralim012 will send you a message on this to see if you have any ideas about how to resolve.

![image](https://user-images.githubusercontent.com/69303397/204713938-b7dce85b-5ac4-403f-86d4-767aa79c9605.png)